pages/app-layout/utils/content-blocks.tsx

```diff
@@ -66,23 +66,18 @@ export function Navigation() {
 
 export function Notifications() {
   const [visible, setVisible] = useState(true);
-  if (!visible) {
-    return null;
+  const notifications: Array<FlashbarProps.MessageDefinition> = [];
+  if (visible) {
+    notifications.push({
```

Comment:
Why do you push an element to the array on the rendering condition? Why not directly provide an array with a single item based on `visible`? At least it would be more readable

Reply:
Do you want something like this

```js
return <Flashbar items={visible ? [demoNotification] : []}>
```

Reply:
yes

Reply:
Done